pandas/core/generic.py

```diff
@@ -2646,6 +2646,10 @@ def to_sql(
         con : sqlalchemy.engine.Engine or sqlite3.Connection
             Using SQLAlchemy makes it possible to use any DB supported by that
             library. Legacy support is provided for sqlite3.Connection objects.
+
```

Comment:
See my comments in the PR - thanks!

Reply:
Thanks for the feedback - new pull request here: https://github.com/pandas-dev/pandas/pull/29373/commits/f080f98a74303af3fa6fedd7c05d4ed19a806bd5